data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -305,14 +300,6 @@ private static boolean hasDeadLetterSink(final EgressConfig egressConfig) {
     return !(egressConfig == null || egressConfig.getDeadLetter().isEmpty());
   }
 
-  private static RecordDispatcherListener getOffsetManager(final DeliveryOrder type, final KafkaConsumer<?, ?> consumer,
-                                                           Consumer<Integer> commitHandler) {
-    return switch (type) {
-      case ORDERED -> new OrderedOffsetManager(consumer, commitHandler);
-      case UNORDERED -> new UnorderedOffsetManager(consumer, commitHandler);
-    };
-  }
-
   private static AbstractVerticle getConsumerVerticle(final DeliveryOrder type,
```

Comment:
So, one Offset mgr is used for the (Un)Ordered Consumers - ?

Reply:
Yes, there is no need to keep 2 separate components and implement the same commit interval logic in 2 places.

The offset tracker overhead in the ordered case is constant.